src/main/java/de/teamlapen/vampirism/core/tags/ModItemTags.java

```diff
@@ -1,45 +1,56 @@
 package de.teamlapen.vampirism.core.tags;
 
 import de.teamlapen.vampirism.api.util.VResourceLocation;
+import net.minecraft.resources.ResourceLocation;
 import net.minecraft.tags.ItemTags;
 import net.minecraft.tags.TagKey;
 import net.minecraft.world.item.Item;
 import org.jetbrains.annotations.NotNull;
 
 public class ModItemTags {
-    public static final TagKey<Item> CROSSBOW_ARROW = tag("crossbow_arrow");
-    public static final TagKey<Item> HUNTER_INTEL = tag("hunter_intel");
-    public static final TagKey<Item> PURE_BLOOD = tag("pure_blood");
-    public static final TagKey<Item> GARLIC = common("crops/garlic");
-    public static final TagKey<Item> HOLY_WATER = tag("holy_water");
-    public static final TagKey<Item> HOLY_WATER_SPLASH = tag("holy_water_splash");
-    public static final TagKey<Item> CURSEDEARTH = tag("cursed_earth");
+    public static final TagKey<Item> CURSED_EARTH = tag("cursed_earth");
     public static final TagKey<Item> DARK_SPRUCE_LOG = tag("dark_spruce_log");
     public static final TagKey<Item> CURSED_SPRUCE_LOG = tag("cursed_spruce_log");
-    public static final TagKey<Item> HEART = tag("heart");
-    public static final TagKey<Item> APPLICABLE_OIL_SWORD = tag("applicable_oil/sword");
-    public static final TagKey<Item> APPLICABLE_OIL_PICKAXE = tag("applicable_oil/pickaxe");
-    public static final TagKey<Item> APPLICABLE_OIL_ARMOR = tag("applicable_oil/armor");
+
     public static final TagKey<Item> DARK_STONE = tag("dark_stone");
     public static final TagKey<Item> DARK_STONE_BRICKS = tag("dark_stone_bricks");
     public static final TagKey<Item> POLISHED_DARK_STONE = tag("polished_dark_brick");
     public static final TagKey<Item> COBBLED_DARK_STONE = tag("cobbled_dark_brick");
     public static final TagKey<Item> DARK_STONE_TILES = tag("dark_brick_tiles");
+
     public static final TagKey<Item> NO_SPAWN = tag("no_spawn");
     public static final TagKey<Item> VAMPIRE_SPAWN = tag("vampire_spawn");
+
+    public static final TagKey<Item> PURE_BLOOD = tag("pure_blood");
+    public static final TagKey<Item> GARLIC = common("crops/garlic");
+    public static final TagKey<Item> HEART = tag("heart");
+
+    public static final TagKey<Item> HOLY_WATER = tag("holy_water");
+    public static final TagKey<Item> HOLY_WATER_SPLASH = tag("holy_water_splash");
+
+    public static final TagKey<Item> HUNTER_INTEL = tag("hunter_intel");
     public static final TagKey<Item> VAMPIRE_BEACON_PAYMENT_ITEM = tag("vampire_beacon_payment_item");
-    public static final TagKey<Item> HEART_SEEKER = tag("swords/heart_seeker");
-    public static final TagKey<Item> HEART_STRIKER = tag("swords/heart_striker");
+
     public static final TagKey<Item> VAMPIRE_SLAYER_ITEMS = tag("enchantable/vampire_slayer");
     public static final TagKey<Item> CROSSBOW_ENCHANTABLE = tag("enchantable/crossbow");
 
+    public static final TagKey<Item> APPLICABLE_OIL_SWORD = tag("applicable_oil/sword");
```

Comment:
Same here for the whole file. Please refrain from doing such refactoring in the PR where you add or remove some fields. That makes it hard to find what you actually changed.